specs/core/core-shapes.ttl

```diff
@@ -446,6 +460,14 @@
                           oslc:valueType           oslc:Resource ;
                           dcterms:description      "For object properties, the URI of a shape resource that describes the object resource."^^rdf:XMLLiteral
                         ] ;
+        oslc:property   [ a                        oslc:Property ;
```

Comment:
What is this? Could you please clearly state all major changes in the PR description?